style.css

```diff
@@ -100,3 +100,255 @@ html {
 .navbar-list-item a {
   color: #fff;
 }
+
+.project {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+}
+
+.underlined {
+  border-bottom: 1px solid #212121;
+}
+
+.projects-heading1 h3 {
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  color: #fff;
+  text-align: left;
+  font-weight: 400;
+  margin-top: 5rem;
+  margin-left: 1rem;
+}
+
+.project .underlined {
+  color: #1d1a19;
+  font-size: 40px;
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+  margin: 3rem 0;
+  text-align: center;
+  padding: 2rem;
+}
+
+.project-heading {
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 400;
+}
+
+.project-item-img {
+  width: 100%;
+}
+
+.project-item-img img {
+  width: 100%;
+  object-fit: cover;
+}
+
+.project-list {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+}
+
+.project-disc {
+  margin-top: 2.4rem;
+}
+
+.project-disc-main {
+  color: #091e42;
+  font-family: 'Inter', serif;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.project-btns {
+  display: flex;
+  gap: 15px;
+}
+
+.project-btns-list {
+  font-weight: 500;
+  font-size: 15px;
+  padding: 1rem;
+  margin: 1rem 0;
+  border: 0.1rem solid #8993a4;
+  background: #fff;
+  color: #091e42;
+  font-family: "Inter", serif;
+}
+
+.project-btn {
+  padding: 1rem 2rem;
+  background-color: #ff6b00;
+  color: #fff;
+  font-family: "Inter", serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 2.4rem;
+  border: none;
+  cursor: pointer;
+  transition: 0.3s all;
+}
```

Comment:
- [ ] Kindly implement the required button styling: Active, Hover and Pressed (see design :point_down: )

![image](https://user-images.githubusercontent.com/99715640/213825507-4ca1c6a4-47be-4630-af94-d61f72db279e.png)